api/disruption_kind.go

```diff
@@ -23,7 +24,7 @@ type DisruptionKind interface {
 
 type DisruptionArgs struct {
 	AllowedHosts         []string
-	TargetContainerIDs   []string
+	TargetContainers     map[string]string
```

Comment:
Changed the logic to transmitting the id of the targeted containers to transmit the names instead: we need to keep the names only so that we can update the ids in case of a container restart